### PR TITLE
[EOC-427] Use raw imports of scss files inside component file

### PR DESCRIPTION
**Pull request by alemikolo**

### CODE REVIEW

File: client/src/common/components/Forms/ErrorMessage.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './ErrorMessage.scss';
```
Comment (Ciunkos) [inline client/src/common/components/Forms/ErrorMessage.jsx]: Please split external and internal import groups.

File: client/src/common/components/Form.jsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { FormattedMessage, injectIntl } from 'react-intl';
 
 import { ListType } from 'modules/list/consts';
 import { IntlPropType } from 'common/constants/propTypes';
+import './Form.scss';
```
Comment (Ciunkos) [inline client/src/common/components/Form.jsx]: Please split external and internal import groups.

File: client/src/common/components/Forms/TextInput.jsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import './TextInput.scss';
```
Comment (Ciunkos) [inline client/src/common/components/Forms/TextInput.jsx]: Please split external and internal import groups.

File: client/src/common/components/Forms/Textarea.jsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import './Textarea.scss';
```
Comment (Ciunkos) [inline client/src/common/components/Forms/Textarea.jsx]: Please split external and internal import groups.

File: client/src/common/components/ListViewItem.jsx [hunk 5]
```diff
@@ -8,6 +8,7 @@ import Preloader, { PreloaderSize } from 'common/components/Preloader';
 import { ListType } from 'modules/list/consts';
 import { ColorType, Routes } from 'common/constants/enums';
 import { IntlPropType } from 'common/constants/propTypes';
+import './ListViewItem.scss';
```
Comment (Ciunkos) [inline client/src/common/components/ListViewItem.jsx]: Please split external and internal import groups.

File: client/src/common/components/Members/components/InviteNewUser.jsx [hunk 6]
```diff
@@ -6,6 +6,7 @@ import PendingButton from 'common/components/PendingButton';
 import { PreloaderTheme } from 'common/components/Preloader';
 import { PROJECT_NAME } from 'common/constants/variables';
 import { IntlPropType } from 'common/constants/propTypes';
+import './InviteNewUser.scss';
```
Comment (Ciunkos) [inline client/src/common/components/Members/components/InviteNewUser.jsx]: Please split external and internal import groups.

File: client/src/common/components/Members/components/MemberRole.jsx [hunk 7]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
+import './MemberRole.scss';
```
Comment (Ciunkos) [inline client/src/common/components/Members/components/MemberRole.jsx]: Please split external and internal import groups.

File: client/src/common/components/Members/components/MembersForm.jsx [hunk 8]
```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames';
 import { injectIntl, FormattedMessage } from 'react-intl';
 import _isEmpty from 'lodash/isEmpty';
 import _trim from 'lodash/trim';
+import './MembersForm.scss';
```
Comment (Ciunkos) [inline client/src/common/components/Members/components/MembersForm.jsx]: Please split external and internal import groups.

File: client/src/modules/list/components/FilterBox.scss [hunk 9]
```diff
@@ -21,7 +23,7 @@
     padding: four-by(2) four-by(4) four-by(2) four-by(2);
     border: 0;
     border-radius: 0;
-    background-image: url('../../assets/images/chevron-down-solid.svg');
+    background-image: url('../../../assets/images/chevron-down-solid.svg');
```
Comment (Ciunkos) [inline client/src/modules/list/components/FilterBox.scss]: Why this change? Was this broken before?

File: client/src/modules/list/components/Items/ListItem/ListItem.scss [hunk 10]
```diff
@@ -17,14 +19,14 @@
...
-        background-image: url('../../assets/images/check-solid.svg');
+        background-image: url('../../../../../assets/images/check-solid.svg');
         background-size: 18px;
         background-position: center;
       }
 
       &:hover {
         &::after {
-          background-image: url('../../assets/images/undo-alt-solid.svg');
+          background-image: url('../../../../../assets/images/undo-alt-solid.svg');
```
Comment (Ciunkos) [inline client/src/modules/list/components/Items/ListItem/ListItem.scss]: Please inspect is some of the assets can be moved nearby the components that use tham so the path will be up to two levels up.

File: client/src/modules/user/AuthBox/AuthBox.scss [hunk 11]
```diff
@@ -26,7 +28,7 @@
     box-sizing: border-box;
     height: 50%;
     padding: four-by(4);
-    background: url('../../assets/images/auth-intro.jpg') center;
+    background: url('../../../assets/images/auth-intro.jpg') center;
```
Comment (Ciunkos) [inline client/src/modules/user/AuthBox/AuthBox.scss]: This may be moved to AuthBox so it can be imported like `./auth-intro.jpg`.

File: webpack.config.js [hunk 12]
```diff
@@ -33,10 +33,15 @@ module.exports = {
...
-          'css-loader', // translates CSS into CommonJS
-          'postcss-loader', // for autoprefixer use
-          'sass-loader' // compiles Sass to CSS, using Node Sass by default
+          'style-loader',
+          'css-loader',
+          'postcss-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              includePaths: ['./client/src/app/styles']
```
Comment (Ciunkos) [inline webpack.config.js]: Is this needed?

File: client/src/common/components/Form.jsx (hunk 2 above)
Comment (alemikolo) [reply client/src/common/components/Form.jsx]: Here imports are split. `import { IntlPropType } from 'common/constants/propTypes';` is our internal import

File: client/src/common/components/ListViewItem.jsx (hunk 5 above)
Comment (alemikolo) [reply client/src/common/components/ListViewItem.jsx]: Here imports are split. `import { IntlPropType } from 'common/constants/propTypes';` is our internal import

File: client/src/common/components/Members/components/InviteNewUser.jsx (hunk 6 above)
Comment (alemikolo) [reply client/src/common/components/Members/components/InviteNewUser.jsx]: Here imports are split. `import { IntlPropType } from 'common/constants/propTypes';` is our internal import
